src/VisualStudio/VisualStudioDiagnosticsToolWindow/OptionPages/ForceLowMemoryMode.cs

```diff
@@ -12,31 +10,36 @@
 
 namespace Roslyn.VisualStudio.DiagnosticsWindow.OptionsPages
 {
-    internal sealed partial class ForceLowMemoryMode
+    internal sealed class ForceLowMemoryMode
     {
-        private readonly IOptionService _optionService;
-        private MemoryHogger _hogger;
+        private const string FeatureName = "ForceLowMemoryMode";
+
+        public static readonly Option2<bool> Enabled = new(FeatureName, "Enabled", defaultValue: false);
+        public static readonly Option2<int> SizeInMegabytes = new(FeatureName, "SizeInMegabytes", defaultValue: 500);
```

Comment:
I am not familiar with how ForceLowMemoryMode is used. I did notice these options previously had a storage location. Are they not intended to persist between sessions? Or be configurable outside of the open session?

Reply:
My impression is that it's only controlled from the Diagnostics Tool Window. But i can imagine someone wanting to set a reg key for automated testing e.g. I'll add the storage location back.